assets/js/modules/analytics/components/common/AcquisitionPieChart.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * AcquisitionPieChart component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __, _x } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import PreviewBlock from '../../../../components/preview-block';
+import GoogleChart from '../../../../components/google-chart';
+import Link from '../../../../components/link';
+import { getSiteKitAdminURL } from '../../../../util';
+import { extractAnalyticsDataForTrafficChart } from '../../util';
+
+const GOOGLE_CHART_PIE_SETTINGS = {
+	chartArea: {
+		width: '100%',
+		height: '100%',
+	},
+	backgroundColor: 'transparent',
+	height: 250,
+	legend: {
+		alignment: 'center',
+		textStyle: {
+			color: '#5b5b61',
+			fontSize: 12,
+		},
+	},
+	slices: {
+		0: { color: '#178EC5' },
+		1: { color: '#54B23B' },
+		2: { color: '#EB5729' },
+		3: { color: '#ECED33' },
+		4: { color: '#34CBE3' },
+		5: { color: '#82E88E' },
+	},
+	title: null,
+	width: '100%',
+};
+
+function AcquisitionPieChart( { data, source } ) {
+	if ( ! data ) {
+		return <PreviewBlock width="282px" height="282px" shape="circular" />;
+	}
+
+	return (
+		<div className="googlesitekit-chart googlesitekit-chart--pie">
+			<GoogleChart
+				data={ extractAnalyticsDataForTrafficChart( data ) }
+				options={ GOOGLE_CHART_PIE_SETTINGS }
+				chartType="pie"
+				id="overview-piechart"
+				loadHeight={ 205 }
+			/>
+
+			{ source && (
+				<div className="googlesitekit-chart__source">
+					{ __( 'Source:', 'google-site-kit' ) }
+					{ ' ' }
+					<Link href={ getSiteKitAdminURL( 'googlesitekit-module-analytics' ) } inherit>
+						{ _x( 'Analytics', 'Service name', 'google-site-kit' ) }
+					</Link>
```

Comment:
Technically we should be combining these strings and using something like `sprintf()` and `dangerouslySetInnerHTML()` with `sanizeHTML()` for this (see an example at https://github.com/google/site-kit-wp/blob/5c342cf4e251fd39e3dd70c9a7bb2a29ac13e771/assets/js/components/error-text.js#L40-L63).

Separating these two strings with a space and placing them in this order works fine in English, but that might not be the case for every locale. True, we lose the ability to use a `<Link>` component and need to use a regular `<a>` instead, but I think in this case it's fine and we can improve the localisation accuracy if we do.

We'd want something like:

```js
sprintf(
	/* translators: %1$s: URL to Analytics Module page in Site Kit Admin, %2$s: Analytics (Service Name) */
	__( 'Source: <a class="googlesitekit-cta-link googlesitekit-cta-link--inherit" href="%1$s">%2$s</a>', 'google-site-kit' ),
	getSiteKitAdminURL( 'googlesitekit-module-analytics' ),
	_x( 'Analytics', 'Service name', 'google-site-kit' ),
);
```

It's not the most lovely approach, but it's better for translations…

Reply:
Yes, agree, it's a good improvement. Updated.